### DISCUSSION TITLE
How to include userId in req body?

### DISCUSSION THREAD

**Discussion by uenlkr4e** [hagopj13/node-express-boilerplate#214] [Q&A]
Hi, 
is it possible to get the userId in req body in controllers for "user is the owner of ..." checks? 
or is there a way to pull it from the access token on controller side? 

i want to make sure the user can only delete his log by making a "user is the owner of this log" check using the userId in token and comparing it with the userId in log entity. 

any ideas? 
thanks in advance

**Comment by alibabayev0** [comment]
Hi, You have req.user in middleware when u check jwt, like in your controller or service you can check it by req.user to get who sent request.
